versioned_docs/version-8.7/apis-tools/camunda-api-rest/camunda-api-rest-overview.md

```diff
@@ -47,19 +43,30 @@ spring.servlet.multipart.max-request-size=4MB
 
 For example, if you increase the `maxMessageSize` to 10MB, increase these property values to 10MB as well.
 
+### Query API
+
+:::danger
+Query API endpoints do not currently support [resource authorizations][], and can be used to expand user access to restricted resources. If you use resource permissions, allowing public access to those endpoints is not recommended.
+:::
+
+All Query API endpoints contain an `(alpha)` declaration. Those endpoints are not accessible by default in Camunda 8 clusters.
+
+You can enable the [alpha feature][] search endpoints by setting either the configuration property `camunda.rest.query.enabled` to `true`,
+or the environment variable `CAMUNDA_REST_QUERY_ENABLED` to `true`.
+
 ## API Key Attributes
 
 OpenAPI key attributes have a `key` suffix, and they serve as the technical unique identifier for entities, such as
 `processDefinitionKey` and `tenantKey`, with `correlationKey` being the only exception. Those attributes are of type
-`string`. In previous versions, key attributes were defined as `integer (int64)`.
+`string` by default. In previous versions, key attributes were defined as `integer (int64)` by default.
 
 To support a gradual change and provide users with enough time, we have introduced new content types in our API
 specification. With these content types, users can specifically tell our API to accept and return `integer (int64)` or
 `string` key attributes.
 
 :::info Deprecation
-Keep in mind that objects with key attributes as `integer (int64)` are marked deprecated, and **will be removed
-in a future release**.
+Objects with key attributes as `integer (int64)`, and the content types associated with them, are deprecated and **will be removed
+in the next minor release**.
```

Comment:
Edited to remove filler phrase, and add precision on what's being removed and when.

Reply:
Good idea 👍